inc/common/purge.php

```diff
@@ -139,7 +139,9 @@ function rocket_clean_post( $post_id ) {
 	// Add Post Type archive
 	if ( 'post' !== $post->post_type ) {
 	    if ( $post_type_archive = get_post_type_archive_link( get_post_type( $post_id ) ) ) {
-	    	array_push( $purge_urls, $post_type_archive );
+	    	array_push( $purge_urls, trailingslashit( $post_type_archive ) . 'index.html' );

```

Comment:
To avoid to use 3 times the same function, we can do before the first array_psuh:
$post_type_archive = trailingslashit( $post_type_archive );